apps/skilavottord/web/screens/AccessControl/AccessControl.tsx

```diff
@@ -276,6 +303,7 @@ const AccessControl: FC = () => {
                     <Text variant="eyebrow">{t.tableHeaders.partner}</Text>
                   </HeadData>
                   <HeadData></HeadData>
+                  <HeadData></HeadData>
```

Comment:
I believe this would look much better if you'd combine these two (edit/delete) into a utility button (or something similar) which would open a menu to chose from actions, in this case, either edit or delete which pops up a different modal.